examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -67,6 +67,8 @@ class ModelCommand : public CHIPCommand
 
     virtual CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex) { return CHIP_ERROR_BAD_REQUEST; };
 
+    virtual void ClearICDEntry(const chip::ScopedNodeId & nodeId);
```

Comment:
This should not be virtual.

Reply:
https://github.com/project-chip/connectedhomeip/pull/33690
follow-up here